lib/tasks/migrate.js

```diff
@@ -56,6 +56,12 @@ module.exports = function runMigrations(context) {
                 message: 'Invalid database username or password',
                 environment: context.instance.system.environment
             });
+        } else if (error.stdout && error.stdout.match(/npm install sqlite3 \-\-save/)) {
+            // We check stdout because knex outputs to stdout on this particular error
+            error = new errors.SystemError(
```

Comment:
https://docs.ghost.org/v1.0.0/docs/troubleshooting#section-sqlite3-install-failure

If you have a second to write out the `ghost config`  command that would switch a user from sqlite3 to mysql that'd be super handy